src/assets/icons/Account.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+function Account() {
+  return (
+    <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
It is worth to explore about patternfly react `Icon` component.

Reply:
I could not find these specific icons in the patternfly icons